### PR TITLE
Bookmarklet for new links

### PR DESCRIPTION
**Pull request by davidjbeveridge**
Adds a bookmarklet link in the header, which allows admins to quickly bookmark
interesting things they find while browsing.

Hope you like it.

### CODE REVIEW

**Comment by rilian** [comment]
nice idea - i'll test soon

**Comment by rilian** [comment]
can you please demonstrate how this bookmarklet may be used ? Also how it can be used outside of Newsletter site ?

This is what happens when I'm clicking on Bookmarklet in menu
s 2015-12-22 at 21 20 35

File: app/controllers/links_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def index
   end
 
   def new
-    @link = Link.new
+    @link = Link.new(new_link_params)
```
Comment (rilian) [inline app/controllers/links_controller.rb:9]: I believe you can reuse `create_link_params` here

Comment (davidjbeveridge) [inline app/controllers/links_controller.rb:9]: I specifically wanted to disallow `issue_id` and `manager_id`.

**Comment by davidjbeveridge** [comment]
I'll add some documentation for the bookmarklet.

**Comment by davidjbeveridge** [comment]
@rilian I've added some docs in [`docs/bookmarklet.md`](https://github.com/davidjbeveridge/newsletter/blob/feature/bookmarklet-for-new-links/docs/bookmarklet.md); I hope that explains the intent/usage a bit better.

File: app/controllers/links_controller.rb (hunk 1 above)
Comment (rilian) [inline app/controllers/links_controller.rb:9]: okay

**Comment by rilian** [comment]
this makes sense now - thanks!

**Comment by sudodoki** [comment]
:+1:

**Pull request merged**